dotcom-rendering/src/components/Nav/ExpandedMenu/VeggieBurgerMenu.tsx

```diff
@@ -77,9 +78,15 @@ const veggieBurgerStyles = (display: ArticleDisplay) => css`
 	position: absolute;
 	border: 0;
 	border-radius: 50%;
+	z-index: 1;
 
 	right: 5px;
 	bottom: 58px;
+
+	${between.mobileMedium.and.tablet} {
+		${getZIndex('burger')}
```

Comment:
[`decideIndex`](![image](https://github.com/guardian/dotcom-rendering/assets/19683595/a0e77887-2392-4294-9110-f77a78855c0a)) method gives `z-index: 22` which gives priority to the burger and serves our purpose.

![image](https://github.com/guardian/dotcom-rendering/assets/19683595/6a0ec806-8d1f-47cb-9dc8-43e8c4bad441)